content/millennium/r4/financial/charge-item.md

```diff
@@ -69,6 +69,43 @@ All URLs for custom extensions are defined as `https://fhir-ehr.cerner.com/r4/St
  `revenue-code`         | None (contains nested extensions) | The type of revenue or cost center providing the product and/or service.
  `unit-price`           | [`Money`]                         | The price of a single unit for the resource.
 
+## Search
+
+<%= beta_tag(action: true) %>
+
+Search for ChargeItems that meet supplied query parameters:
+
+    GET /ChargeItem?:parameters
+
+### Authorization Types
+
+<%= authorization_types(provider: true, patient: true, system: true) %>
+
+### Parameters
+
+ Name                  | Required?  | Type          | Description
+-----------------------|------------|---------------|-------------------------------------------------------
+ `_id`                 | Yes        | [`token`]     | The logical resource id associated with the resource.
```

Comment:
Let's change the description to "The logical resource id associated with the ChargeItem."

Reply:
updated here https://github.com/cerner/fhir.cerner.com/pull/653/commits/baac66e5788771abe9ea6e9e8ae95e815d012d57#diff-16d4f8f74626c62d57a8b71d6be699f60e72fffed8636bc32e6654d4252b3ebbR88

Reply:
+1 to the changes.